sdk/python/v1beta1/README.md

```diff
@@ -1,125 +1,53 @@
-# Kubeflow Katib SDK
+Katib is a Kubernetes-native project designed for automated machine learning. It focuses on hyperparameter tuning, neural architecture search (NAS), and early stopping of machine learning experiments. Katib supports various machine learning frameworks and languages, making it versatile for different applications.
 
-Python SDK for Kubeflow Katib
+## Requirements
+**Python**: >= 3.8
 
-## Requirements.
+The Katib Python SDK follows the [Python release cycle](https://devguide.python.org/versions/#python-release-cycle) for supported Python versions.
 
-Python >= 3.8
+## Installation
+You can install the Katib Python SDK using either pip or setuptools.
 
-Katib Python SDK follows [Python release cycle](https://devguide.python.org/versions/#python-release-cycle)
-for supported Python versions.
-
-## Installation & Usage
-
-### pip install
+### Using pip
+To install the SDK via pip, run:
 
 ```sh
 pip install kubeflow-katib
 ```
 
-Then import package:
+After installation, you can import the package in your Python code:
 
 ```python
 from kubeflow import katib
 ```
 
-### Setuptools
+### Using Setuptools
+To install via Setuptools, clone the repository and run:
 
 Install via [Setuptools](http://pypi.python.org/pypi/setuptools).
 
 ```sh
 python setup.py install --user
 ```
+Alternatively, use sudo to install the package for all users:
 
-(or `sudo python setup.py install` to install the package for all users)
+```sh
+sudo python setup.py install
+```
 
 ### Publish new SDK version to PyPi
 
-Our SDK is located in [`kubeflow-katib` PyPi package](https://pypi.org/project/kubeflow-katib/).
-Katib Python SDK is published as part of the Katib patch releases.
-You can check the release process [here](../../../scripts/v1beta1/release.sh).
-For each Katib patch release, we upload a new SDK version to the PyPi.
-The SDK version is equal to the Katib version.
+The Katib Python SDK is released as part of Katib's patch releases. For each patch release, a new version of the SDK is uploaded to PyPi. The SDK version corresponds directly to the Katib version.
+
+You can find the Katib Python SDK in the [`kubeflow-katib` package on PyPi](https://pypi.org/project/kubeflow-katib/).
+
+For detailed information on the release process, please refer to the [release script](https://github.com/shivas1516/katib/blob/master/scripts/v1beta1/release.sh) in the Katib repository.
```

Comment:
needs to be replaced with main repo

Reply:
updated in the recent push, check and let me know if addition changes sir